docs/manuals/files.tex

```diff
@@ -2510,7 +2510,7 @@ \subsection{Boundary Conditions: Pressure}
 change according to any cycle.  Instead, time dependence is handled by
 evaluating at the time the boundary condition value is desired.
 The solution is specified by using a predefined function (choices are described
-below).  NOTE: These last six types of boundary condition input is for
+below).  The choice {\bf GroundwaterFlow} turns on groundwater interactions with a deep aquifer as described in \cite{Rahman2018}. The aquifer is set with the Specific Yield and Thickness parameters, detailed below.   NOTE: These last seven types of boundary condition inputs are for
```

Comment:
There are some limitations noted in a comment on github about only supporting flat bottom.   At a minimum those should be documented.